Configuration/RealmSwift/RealmSwift.xcconfig

```diff
@@ -10,3 +10,5 @@ DEFINES_MODULE = YES;
 INFOPLIST_FILE = RealmSwift/RealmSwift-Info.plist;
 PRODUCT_NAME = RealmSwift;
 PRODUCT_BUNDLE_IDENTIFIER = io.realm.RealmSwit;
+
+OTHER_SWIFT_FLAGS = -D BUILDING_REALM_SWIFT;
```

Comment:
Doesn't this need to be defined in the podspec as well?

If that's difficult then inverting the condition would avoid that.

Reply:
I assume it would. I'm quit surprised that the CocoaPods jobs succeeded without it!

Reply:
It succeeded because the Swift compiler only emits a warning about the redundant `import`. I went ahead and inverted the condition in e1b6c8d anyway.